.github/workflows/publish.yml

```diff
@@ -1,14 +1,27 @@
+# .github/workflows/publish.yml
 name: Publish to pub.dev
 
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
+    - 'v[0-9]+.[0-9]+.[0-9]+*' # tag pattern on pub.dev: 'v{{version}'
 
+# Publish using custom workflow
 jobs:
   publish:
     permissions:
       id-token: write # Required for authentication using OIDC
-    uses: dart-lang/setup-dart/.github/workflows/publish.yml@v1
-    # with:
-    #   working-directory: path/to/package/within/repository
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4     
+      
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+          
+      - name: Install dependencies
+        run: flutter pub get
+        
+      - name: Publish
+        run: dart pub publish --force
```

Comment:
Can we add a check for when the publishing would fail or other errors would occur? Or will this also fail with the --force flag?

Reply:
There are checks that have to be bypassed by the flag. The command will always ask if you're sure you want to publish because publishing is permanent. If it actually fails, this action fails. If it shows warnings it still does it (rip pub points).